src/sentry/static/sentry/app/components/charts/chartZoom.jsx

```diff
@@ -103,20 +105,22 @@ class ChartZoom extends React.Component {
     //
     // Parent container can use this to change into a loading state before
     // URL parameters are changed
-    callIfFunction(this.props.onZoom, {
+    callIfFunction(onZoom, {
       period,
       start: startFormatted,
       end: endFormatted,
     });
 
     this.zooming = () => {
-      updateParams(
```

Comment:
I may change these to be private - I don't think we want to expose these since they do not update the store.

Reply:
So I believe the issue I had with this is that echarts doesn't expose an event when the chart is finished zooming. I can look into it again and see if it has changed since, but would prefer that to be in a follow-up if it is possible

Reply:
So from what I'm seeing this is what is happening:

1) User zooms
2) eCharts starts zoom animation
3) eCharts finishes zoom animation
4) Update URL + store
5) Fetch new data (clears chart)
6) Renders new zoomed in chart based on new absolute date (which is shaped a bit differently than the eCharts local zoom chart shape).

Reply:
Yeah the animation and then reload is unpleasant. Fixing that separately totally makes sense as it is a tricky problem.